src/audio/multiband_drc/multiband_drc.c

```diff
@@ -250,17 +250,7 @@ static int multiband_drc_init(struct processing_module *mod)
 	md->private = cd;
 	cd->multiband_drc_func = NULL;
 	cd->crossover_split = NULL;
-#if CONFIG_IPC_MAJOR_4
```

Comment:
I think I need to drop this change for IPC4. The default should be enabled true, since an old kernel would never enable this switch control.

With new kernel the switch control is set to cached control state right after init, so the processing would be off if requested so. Also in IPC4 this control is not read back to kernel so this won't impact control state in kernel.


--> So I'd just change the comment and explain new and old kernel different and keep enabled = true.